src/Tests/Testing.Common/Testing/CodeVerifier.cs

```diff
@@ -362,12 +362,12 @@ internal static (Document document, ImmutableArray<ExpectedDocument> expectedDoc
 
             for (int i = 0; i < additionalFiles.Length; i++)
             {
-                string documentName = AppendNumberToFileName(options.DocumentName, i + 2);
+                string documentName = Path.GetFileName(additionalFiles[i].Path) ?? AppendNumberToFileName(options.DocumentName, i + 2);
```

Comment:
in case I am providing only a path, it could work erroneously. I do not know how it would behave for cases `Foo/Bar` and `Foo/Bar/`

Reply:
I added validation of file path